components/script/dom/globalscope.rs

```diff
@@ -3312,7 +3317,7 @@ impl GlobalScope {
                                             window.task_manager().gamepad_task_source().queue_with_canceller(
                                                 task!(update_gamepad_connect: move || {
                                                     let gamepad = new_gamepad.root();
-                                                    gamepad.notify_event(GamepadEventType::Connected);
+                                                    gamepad.notify_event(GamepadEventType::Connected, can_gc);
```

Comment:
Let's revert this, since it's inside of a task! macro.

Reply:
sorry for the overlook, let me fix this